README.md

```diff
@@ -6,4 +6,4 @@ Feel free to go wild here.
 
 ## The bikeshed.
 
-The [color of the bikeshed](https://bikeshed.com) is yet to be decided.
+The [color of the bikeshed](https://bikeshed.com) is green with pink stripes.
```

Comment:
OK, very good, that's close to "teal with pink stripes", you are refining this, I like it.